docs/tutorial.rst

```diff
@@ -14,6 +14,17 @@ On macOS and Linux, the easiest way to install CumulusCI is using `homebrew <htt
     $ brew tap SFDO-Tooling/homebrew-sfdo
     $ brew install cumulusci
 
+On Windows 10, the easiest way to install CumulusCI is using ```pipx`` <https://github.com/pipxproject/pipx>`__: 
+
+1. In a new command prompt, run: ``python -m pip install --user pipx``
```

Comment:
This is assuming that the user already has the correct version of Python installed, but we don't tell them how to do that until further down.